src/GitHub.App/Services/GistPublishService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using System.IO;
+using System.Reactive.Linq;
+using GitHub.Api;
+using GitHub.Models;
+using LibGit2Sharp;
+using Octokit;
+
+namespace GitHub.Services
+{
+    [Export(typeof(IGistPublishService))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
+    public class GistPublishService : IGistPublishService
+    {
+        [ImportingConstructor]
+        public GistPublishService()
```

Comment:
Ok, yep - this was just copied from another service without considering that. Will remove.